### PR TITLE
Show checklist once user submit a pull request.

### PR DESCRIPTION
**Pull request by yurisasuke**
This pull request makes sure that users are shown a checklist after submitting a pull request to make sure a pull request meet some criteria.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+<!-- Why is this change required? What problem does it solve? -->
+
+## Preview Link
+<!-- Add a preview link to make it easier for reviewer to review. -->
+## Screenshots (if appropriate)
+
+## Types of changes
+<!-- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Fixing typo 
+- [ ] Documenting a new feature
```
Comment (letzya) [inline .github/pull_request_template.md]:
```suggestion
- [ ] Documenting a new feature (please merge to production)
- [ ] Documentation for future release (please do not merge to production)
- [ ] Something else (please add if needs merging to production or not)
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+## Motivation and Context
+<!-- Why is this change required? What problem does it solve? -->
+
+## Preview Link
+<!-- Add a preview link to make it easier for reviewer to review. -->
+## Screenshots (if appropriate)
+
+## Types of changes
+<!-- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Fixing typo
```
Comment (letzya) [inline .github/pull_request_template.md]:
```suggestion
- [ ] Fixing typo (please merge to production)
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+## Screenshots (if appropriate)
+
+## Types of changes
+<!-- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Fixing typo 
+- [ ] Documenting a new feature
+
+## Checklist
+<!-- Go over all the following points, and put an `x` in all the boxes that apply -->
+- [ ] I have [reviewed the guidelines](../CONTRIBUTING.md) for contributing to this repository.
```
Comment (letzya) [inline .github/pull_request_template.md:32]: please use the ref format `{{< ref "link" >}}

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+- [ ] I have [read the technical guidelines](../CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
```
Comment (letzya) [inline .github/pull_request_template.md:31]: ref link

**Review by letzya — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 3 above)
Comment (letzya) [reply .github/pull_request_template.md:32]: Dont you think it's too much to ask for every time you do a pr?

**Review by letzya — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 3 above)
Comment (yurisasuke) [reply .github/pull_request_template.md:32]: The user will only need to read the guide only on their first commit.In subsequent commits they can just tick without reading as they are already familiar with the guidelines.

Comment (letzya) [reply .github/pull_request_template.md:32]: ok, we can try, just don't think people will tick this at all

**Review by letzya — APPROVED** [review]

**Pull request merged**